src/main/java/com/company/skyproeducationspring/repositories/question/QuestionRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.company.skyproeducationspring.repositories.question;
+
+import com.company.skyproeducationspring.models.Question;
+
+import java.util.ArrayList;
+
+public interface QuestionRepository {
+    Question add(Question question);
+
+    Question remove(Question question);
+
+    boolean contains(Question question);
+
+    ArrayList<Question> getAll();
```

Comment:
```suggestion
   List<Question> getAll();
```
стоит предпочитать абстракции конкретным реализациям

Reply:
Исправил